readme.md

```diff
@@ -53,6 +52,36 @@ tf = textr({ locale: 'ru'})
 tf('Hello  "world"...\n'); // Hello «world»…
 ```
 
+## CLI
+
+Also, there is a [CLI package][cli] for running Textr under your terminal. You can install it typing:
+
+```
+npm install --global textr-cli
+```
+
+After installation, you can process files using `textr`-command, like this:
+
+```sh
+# load from file, transformers required through few -t flags
+$ textr foo.md -t typographic-quotes -t typographic-quotes
+
+# load from file, transformers required through one -t
+$ textr foo.md -t typographic-single-spaces,typographic-quotes
+
+# load through stdin, iso-locale equals ru, transformers using --transforms
+$ cat foo.md | textr -l ru --transforms=typographic-single-spaces
+
+# load through stdin, write result into file
+$ cat foo.md | textr -o bar.md
+```
+
+Take a look for more [usage examples][examples] and [CLI options][options].
+
+[cli]: https://github.com/denysdovhan/textr-cli
+[examples]: https://github.com/denysdovhan/textr-cli#usage
+[options]: https://github.com/denysdovhan/textr-cli#options
+
```

Comment:
I think base flags will stand unchanged in future, so we can keep it readme. Of course, will be new features but for this case there is line:

> Take a look for **more** [usage examples][examples] and [CLI options][options].

[examples]: https://github.com/denysdovhan/textr-cli#usage
[options]: https://github.com/denysdovhan/textr-cli#options